components/lib/state/src/main/java/mozilla/components/lib/state/ext/StoreExtensions.kt

```diff
@@ -60,10 +62,10 @@ private class LifecycleBoundObserver(
 /**
  * View.OnAttachStateChangeListener implementation to bind an observer to a View.
  */
-private class ViewBoundObserver(
+private class ViewBoundObserver<S : State, A: Action>(
```

Comment:
Since `LifecycleBoundObserver` and `ViewBoundObserver` are both bindings, perhaps they should be called `Binding` instead?